test/ui/cypress/e2e/deployment_driver.cy.js

```diff
@@ -18,38 +11,101 @@ describe('Deployment driver web UI', () => {
     cy.get('@main').contains('Deployment Steps')
     cy.get('@main').get('div.deployProgress').as('statusArea')
     cy.get('@statusArea').get('button.cancelButton').contains('Cancel Deployment')
+
+    cy.get('.pf-c-brand').should('be.visible')
+    cy.get('.pf-c-alert__title')
+      .contains('You currently have a subscription to Ansible Automation Platform')
+    cy.get('p').contains('To manage or setup new subscription, visit the')
```

Comment:
This is too generic, it will look for any `<p>` tag, it needs to be more specific.

Reply:
@tznamena , this is the selector that I could get from cypress. Do you have any good way to replace it?

Reply:
I would look for an element that's more "significant" to the component. But it is ok to leave this one in for now and we can adjust it in future PR since this works for now.